dune-precice/CMakeLists.txt

```
@@ -0,0 +1,33 @@
+# We require version CMake version 3.1 to prevent issues
+# with dune_enable_all_packages and older CMake versions.
+cmake_minimum_required(VERSION 3.1)
+project(dune-precice CXX)
+
+if(NOT (dune-common_DIR OR dune-common_ROOT OR
+      "${CMAKE_PREFIX_PATH}" MATCHES ".*dune-common.*"))
+    string(REPLACE  ${CMAKE_PROJECT_NAME} dune-common dune-common_DIR
+      ${PROJECT_BINARY_DIR})
+endif()
+
+#find dune-common and set the module path
+find_package(dune-common REQUIRED)
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/modules"
+  ${dune-common_MODULE_PATH})
+
+#include the dune macros
+include(DuneMacros)
+
+# start a dune project with information from dune.module
+dune_project()
+
+dune_enable_all_packages()
+
+add_subdirectory(src)
+add_subdirectory(dune)
+add_subdirectory(cmake/modules)
+
+# find precice library
+find_package(precice REQUIRED CONFIG)
```

Does the adapter work with any preCICE version (v1 vs. v2)? If not consider adding a minimum version.
```suggestion
find_package(precice 2.0.0 REQUIRED CONFIG)
```

---

I added this to the dune-precice cmake, as we only want to work with version 2.0.